### PR TITLE
Create step by step migration guide for OpenIddict

### PR DESCRIPTION
**Pull request by gterdem**
Resolves https://github.com/abpframework/abp/issues/13459

### CODE REVIEW

File: docs/en/Migration-Guides/Abp-6_0.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ LeptonX Lite is now being introduced and you can follow the guides below to migr
 
 ## Migrating to OpenIddict
 
-After the [announcement of plan to replace the IdentityServer](https://github.com/abpframework/abp/issues/11989), we have successfully implemented [Openiddict](https://github.com/openiddict/openiddict-core) as a replacement for IdentityServer4 as OpenID-Provider. You can follow [Migrating Identity Server to OpenIddict guide](./IdentityServer_To_OpenIddict.md).
+After the [announcement of plan to replace the IdentityServer](https://github.com/abpframework/abp/issues/11989), we have successfully implemented [Openiddict](https://github.com/openiddict/openiddict-core) as a replacement for IdentityServer4 as OpenID-Provider. You can follow the [Migrating Identity Server to OpenIddict guide](IdentityServer_To_OpenIddict.md) with a sample project and the [IdentityServer to OpenIddict Step by Step Guide](OpenIddict-Step-by-Step.md) for migrating your existing application in details.
```
Comment (johnhbarrett) [inline docs/en/Migration-Guides/Abp-6_0.md]: better grammar: 
"replacement for IdentityServer4 as an OpenID-Provider"

File: docs/en/Migration-Guides/OpenIddict-Step-by-Step.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
+# Migrating from IdentityServer to OpenIddict Step by Step Guide
+
+This guide provides layer-by-layer guidance for migrating your existing application to OpenIddict. Since OpenIddict is only available with ABP v6.0, you need to update your existing application and apply required migrations in order to apply OpenIddict changes.
```
Comment (johnhbarrett) [inline docs/en/Migration-Guides/OpenIddict-Step-by-Step.md]: grammar:
"This guide provides a layer-by-layer...

"you will need to update your existing application...

"in order to apply the OpenIddict changes...

File: docs/en/Migration-Guides/Abp-6_0.md (hunk 1 above)
Comment (gterdem) [reply docs/en/Migration-Guides/Abp-6_0.md]: I couldn't understand. Can you provide the full sentence?

Comment (johnhbarrett) [reply docs/en/Migration-Guides/Abp-6_0.md]: **before:**
After the [announcement of plan to replace the IdentityServer](https://github.com/abpframework/abp/issues/11989), we have successfully implemented [Openiddict](https://github.com/openiddict/openiddict-core) as a replacement for IdentityServer4 as OpenID-Provider.

**after:**
After the [announcement of plan to replace the IdentityServer](https://github.com/abpframework/abp/issues/11989), we have successfully implemented [Openiddict](https://github.com/openiddict/openiddict-core) as a replacement for IdentityServer4 as **an** OpenID-Provider.

Comment (EngincanV) [reply docs/en/Migration-Guides/Abp-6_0.md:23]: I guess, he means to change the "replacement for IdentityServer4 as OpenID-Provider" as "replacement for IdentityServer4 as **an** OpenID-Provider".

Comment (johnhbarrett) [reply docs/en/Migration-Guides/Abp-6_0.md]: 'OpenID-Provider' is a noun in this context - so you should use "an" as the pro-noun

Comment (gterdem) [reply docs/en/Migration-Guides/Abp-6_0.md]: Thanks!

**Comment by gterdem** [comment]
@braim23 
Can you check the docs for better grammar, please?

**Review by EngincanV — APPROVED** [review]

**Review by ebicoglu — APPROVED** [review]

**Pull request merged**